packages/suite/src/config/wallet/index.ts

```diff
@@ -1,3 +1,4 @@
-import NETWORKS from './networks';
+import { networksCompatibility } from '@suite-common/wallet-networks-config';
```

Comment:
why not just `networksConfig`? It has info not only related to compatibility.

Reply:
There are two networks object one with new shape and one with legacy, legacy has compatibilty suffix. I did not came up with this name, it was used in networks before only with default export so you were able to import it as `NETWORKS`. I don't want change it, it's not purpose of this PR, in this PR I am just moving things around and doing absolute minimum of necessary changes to code.

Reply:
Oh, so now you are also exporting the original `const networks` too, I see. Previously it was not exported, it was only possible to use the `const compatibility` version of it. In that case, perhaps it worth renaming to `const networksConfig` _(it's not imported so should not require any refactors)_ and `const compatibleNetworksConfig`? `networksCompatibility` sounds like it's data about networks being compatible with something.

Reply:
Yea, I will rename and mark it as deprecated.